src/account/AccountItem.js

```diff
@@ -48,17 +55,17 @@ export default class AccountItem extends React.PureComponent {
     this.props.onRemove(this.props.index);
 
   render() {
-    const { email, realm } = this.props;
+    const { email, realm, index } = this.props;
 
     return (
       <Touchable style={styles.wrapper} onPress={this.handleSelect}>
-        <View style={styles.accountItem}>
+        <View style={[styles.accountItem, (index === 0) && styles.selectedAccountItem]}>
           <View style={styles.details}>
-            <Text style={styles.text}>{email}</Text>
-            <Text style={styles.text}>{realm}</Text>
+            <Text style={[styles.text, (index === 0) && styles.selectedText]}>{email}</Text>
+            <Text style={[styles.text, (index === 0) && styles.selectedText]}>{realm}</Text>
           </View>
           <Button
-            secondary
+            secondary={(index === 0) ? 0 : 1}
```

Comment:
`secondary` is a boolean so we should use `true` and `false` here.

This could actually be simplified to: `secondary={index !== 0}`